clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -8266,6 +8266,37 @@ void Clang::AddClangCLArgs(const ArgList &Args, types::ID InputType,
                                                      << "/kernel";
  }
 
+  if (Args.hasArg(options::OPT__SLASH_vlen_default)) {
+    // Override /vlen= options.
```

Comment:
Shouldn't the last flag take precedence? For example, in `/arch:AVX512 /vlen /vlen=256`, shouldn't 256 win? What does MSVC do?